src/components/LocationPage/LocationPageHeader.style.tsx

```diff
@@ -20,12 +20,12 @@ export const ColoredHeaderBanner = styled(Box)`
 `;
 
 export const Wrapper = styled(Box)<{
-  condensed?: Boolean;
-  headerTopMargin: number;
-  headerBottomMargin: number;
+  $condensed?: Boolean;
```

Comment:
I think this is supposed to be `boolean` instead of `Boolean`, maybe we can fix that as well in this PR?

Reply:
surprised the typechecker didn't freakout about this, hm.